backend/question/questions.py

```diff
@@ -57,10 +57,9 @@ def get_questions_from_series(questionseries_set):
     return [create_question_db(key) for key in keys_all]
 
 
-def create_default_questions(overwrite = False):
+def create_default_questions(Question=Question, Choice=Choice, QuestionGroup=QuestionGroup):
```

Comment:
It's very unusual to use upper/camelcase argument names. Better to give the arguments a descriptive snake-case name, e.g., "question_model". Could we actually circumvent passing the model along if we rewrite this function as a member of the `Question` model? Then we could get `Choice` and `QuestionGroup` via the foreign key lookups, and wouldn't need to be concerned with the "version" of the model/object we are using.